test/vm.py

```diff
@@ -103,3 +106,114 @@ def run(self, cmd, user, password):
             output.write(out)
         exit_status = stdout_f.channel.recv_exit_status()
         return exit_status, output.getvalue()
+
+
+class AWS(VM):
+
+    _ssh_port = 22
+    _instance_type = "t3.medium"  # set based on architecture when we add arm tests
+
+    def __init__(self, ami_id):
+        self._ami_id = ami_id
+        self._ec2_instance = None
+        self._ec2_security_group = None
+        self._ec2_resource = boto3.resource("ec2", region_name=AWS_REGION)
+
+    def start(self):
+        sec_group_ids = []
+        if not self._ec2_security_group:
+            self._set_ssh_security_group()
+        sec_group_ids = [self._ec2_security_group.id]
+        try:
+            print(f"Creating ec2 instance from {self._ami_id}")
```

Comment:
(nitpick) if we use `self._log()` here we will be pepared for the time when we have better logging for this :) (and in the other places that now use print())

Reply:
Not a nitpick.  You're absolutely right and we should.

Reply:
Done.